tv.html

```diff
@@ -0,0 +1,239 @@
+        More from brian tome
+      </h2>
+      <div class="col-sm-6 push-bottom">
+        <crds-portrait-card href="https://www.crossroads.net/media/tags/garage-bible-study/" theme="default"
+          lead="Watch Now" title="garage bible study"
+          image-src="//crds-media.imgix.net/36vvUyQjYYKRzs5gghFNKb/fccad33eb2398fef7160b4092b25f06e/bt-garage.png">
+        </crds-portrait-card>
+      </div>
+      <div class="col-sm-6 push-bottom">
+        <crds-portrait-card href="https://crossroads.net/media/podcasts/the-aggressive-life-with-brian-tome"
+          theme="default" lead="Listen Now" title="Agressive life podcast"
+          image-src="//crds-media.imgix.net/5APtNFVXDhx8uTbHpUc5sD/f2bb379a5b3ad54681f6a3ec37ff7bc5/bt-agressive-life-with-logos.png">
+        </crds-portrait-card>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="jumbotron overflow-hidden flush-bottom stream-bg" data-optimize-bg-img>
+  <div class="bg-overlay"></div>
+  <div class="container soft-ends push-ends">
+    <div class="row df-screen-sm  soft-ends push-ends">
+      <div class="col-sm-6 text-left soft-ends push-ends">
+        <h2 class="text-white">Streaming Platforms</h2>
+        <p class="text-white">The Crossroads TV App brings church into your living room. It’s the best way to watch the
+          Crossroads Weekly. Plus it’s full of content your entire family will love like Crossroads Music, Kids’ Club
+          weekly teaching, and Spark Talks.</p>
+      </div>
+      <div class="col-sm-6 ma-ends">
+        <a class="btn btn-white btn-outline tv-btns"
+          href="https://www.amazon.com/dp/B07HRPW8L2?keywords=crossroads+anywhere&qid=1538139831&s=STRING%28skills-store-name%29&sr=1-2-catcorr&ref=sr_1_2"><img
+            src="https://crds-media.imgix.net/34aaswWuopAwfiXFtmqjQv/6900cd357aaf39a65981d08d61d0f8b8/Logo_Amazon_Alexa.png"></a>
+        <a class="btn btn-white btn-outline tv-btns"
+          href="https://channelstore.roku.com/details/9f32b1953ae573adaf019839a8c99093/crossroads"><img
+            src="https://crds-media.imgix.net/31iHeiDmBD3KrDxMHVw4Mo/7d24ef2b66f5d62515905e7266b97708/Logo_Roku.png"></a>
+        <a class="btn btn-white btn-outline tv-btns" href="https://www.amazon.com/Crossroads-Church/dp/B07D83Z1JB"><img
+            src="https://crds-media.imgix.net/3gazTXowti629q6Ai9WW3/a6311faf7f8851b6356e8f598b135443/Logo_Amazongfire.png"></a>
+        <a class="btn btn-white btn-outline tv-btns"
+          href="https://appadvice.com/tv/app/crossroads-anywhere/1029478803"><img class="h-100"
+            src="https://crds-media.imgix.net/5TNLQmLrg05eBQVCEPM8o4/3315e8ce68ac1f9ef75523cb6966a450/Apple_TV_logo.png"></a>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="push-top">
+  <div class="container">
+    <div class="col-sm-4 push-bottom">
+      <crds-portrait-card href="https://www.crossroads.net/about/" theme="default" lead="curious about crossroads?"
```

Comment:
change to `/about` this page lives in all enviroments. Needs to change for all of the other pages that are being link to in this page that fall under this category.